WebHostLib/api/tracker.py

```diff
@@ -0,0 +1,82 @@
+import collections
+import datetime
+
+from typing import Optional
+from flask import abort, jsonify
+
+from Utils import restricted_loads
+from uuid import UUID
+
+from WebHostLib.models import Room
+from WebHostLib.api import api_endpoints
+from WebHostLib.tracker import get_static_room_data
+from WebHostLib import cache
+
+
+@api_endpoints.route('/tracker/<suuid:tracker>')
+@cache.memoize(timeout=60)
+def tracker_data(tracker: UUID):
+    room: Optional[Room] = Room.get(tracker=tracker)
+    if not room:
+        abort(404)
+
+    locations, names, use_door_tracker, player_checks_in_area, \
+        player_location_to_area, precollected_items, games, slot_data, groups = get_static_room_data(room)
+
+    checks_done = {team_number: {player_number: 0
+                                 for player_number in range(1, len(team) + 1) if player_number not in groups}
+                   for team_number, team in enumerate(names)}
+
+    hints = {team: [] for team in range(len(names))}
+
+    finished_players = {team_number: [] for team_number, team in enumerate(names)}
+    if room.multisave:
+        multisave = restricted_loads(room.multisave)
+    else:
+        multisave = {}
+
+    if "hints" in multisave:
+        for (team, slot), slot_hints in multisave["hints"].items():
+            hints[team] = [hint for hint in slot_hints]
+
+    for player in locations:
+        locations[player] = len(locations[player].keys())
+
+    for (team, player), locations_checked in multisave.get("location_checks", {}).items():
+        if player in groups:
+            continue
+        player_locations = locations[player]
+        for location in locations_checked:
+            if location not in player_locations or location not in player_location_to_area[player]:
+                continue
+            checks_done[team][player] += 1
+
+    for (team, player), game_state in multisave.get("client_game_state", {}).items():
+        if player in groups:
+            continue
+        if game_state == 30:
+            finished_players[team].append(player)
```

Comment:
I suggest just returning the game state, so that in the future whenever /ready is fully implemented it can be pulled too